turbo.json

```diff
@@ -7,65 +7,49 @@
       "outputs": ["dist/**", "dist-esm/**", "dist-test/**", "types/**"]
     },
     "build:samples": {
-      "dependsOn": [
-        "^build:samples"
-      ],
       "cache": false
     },
     "build:test": {
       "dependsOn": ["^build:test"],
```

Comment:
Should not depend on dependencies' "build:test", probably should depend on "build"